scene/3d/mesh_instance.cpp

```diff
@@ -1373,8 +963,8 @@ void MeshInstance::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("create_debug_tangents"), &MeshInstance::create_debug_tangents);
 	ClassDB::set_method_flags("MeshInstance", "create_debug_tangents", METHOD_FLAGS_DEFAULT | METHOD_FLAG_EDITOR);
 
-	ClassDB::bind_method(D_METHOD("is_mergeable_with", "other_mesh_instance"), &MeshInstance::is_mergeable_with);
-	ClassDB::bind_method(D_METHOD("merge_meshes", "mesh_instances", "use_global_space", "check_compatibility"), &MeshInstance::merge_meshes, DEFVAL(Vector<Variant>()), DEFVAL(false), DEFVAL(true));
```

Comment:
Doesn't removing the default value for `mesh_instances` break compat?

Reply:
Just refreshing my memory as it is a year since I wrote this(!). But I believe the old default wasn't super useful anyway, as you can't merge a `MeshInstance` with a list containing zero other mesh instances. :grinning: 

Can change this to add the old default though.

Reply:
Ah yes indeed, the new `merge_meshes()` function does report:
```
ERR_FAIL_COND_V(!mis.size(), "Array contains no MeshInstances");
```
If there are no meshes, so having a blank list is no longer a sensible default I think. I don't think this is really compat breaking, as no one should have been using the function like this before, it was more an oversight in the previous code.